docs/pages/build-reference/variables.md

```diff
@@ -241,10 +241,12 @@ The following are two possible alternative approaches, each with different trade
 
 <div style={{marginTop: -20, display: 'block'}} />
 
+> ⚠️  Secrets set on the Expo website or through the EAS CLI will override any secrets set within your eas.json or app.config.js files. 
```

Comment:
you don't set any secrets in eas.json or app.config.js. those are unencrypted env vars

Reply:
![https://media.giphy.com/media/ggu8E0UeH7bINjvacc/giphy-downsized.gif](https://media.giphy.com/media/ggu8E0UeH7bINjvacc/giphy-downsized.gif)

Reply:
just updated, let me know if that works or if you want to see something different.